core/src/no/group15/playmagic/ui/views/MainMenuView.kt

```diff
@@ -35,14 +38,14 @@ class MainMenuView(
 			appContext.screen = GamePresenter(appContext, batch)
 		}
 
-		viewPort.apply()
-		batch.projectionMatrix = viewPort.camera.combined
+		viewport.apply()
+		batch.projectionMatrix = viewport.camera.combined
 		batch.begin()
 		font.draw(
 			batch,
 			glyph,
-			menuWidth / 2 - glyph.width / 2,
-			menuHeight / 2 + glyph.height / 2
+			viewport.worldWidth / 2 - glyph.width / 2,
+			refMenuHeight / 2 + glyph.height / 2
 		)
 		batch.end()
```

Comment:
KTX graphics utilities simplifies this to
```kotlin
import ktx.graphics.*
// Projection matrix will be copied from the camera:
batch.use(camera) {
  font.draw(...)
}
```
Take a look at the example [here](https://github.com/libktx/ktx/tree/master/graphics#usage-examples)